docs/rfcs/XeTile.md

```diff
@@ -516,3 +516,178 @@ With the optimized mapping, the tile_transpose below could be implemented with i
 %10 = load_tile %2  : tile<256x32xf16 #mp_bt> -> vector<256x32xf16>// sg_layout=[32,1], sg_data=[64,32]
 %5  = tile_transpose %10 {#mp_bt #mp_b}: vector<256x32xf16> -> vector<32x256xf16>   // sg_layout=[32,1] ->sg_layout=[8,4]
 ```
+
+## Appendix 2.4 Gemm implementation using cooperative load through shared local memory 
```

Comment:
Maybe 1-2 sentences describing the purpose of this and Appendix 2.5 as we have for other Appendices?

Reply:
added